docs/.vuepress/components/SendTransaction.vue

```diff
@@ -18,18 +18,19 @@ export default {
   },
   methods: {
     sendEthTransaction() {
-      web3.eth.sendTransaction(
-        {
+      ethereum.request({
+        method: 'eth_sendTransaction',
+        params: [{
           from: this.accounts[0],
           to: "0x2f318C334780961FB129D2a6c30D0763d9a5C970",
           value: "0x29a2241af62c0000",
           gas: 21000,
           gasPrice: 20000000000,
-        },
-        (result) => {
+        }]
+      })
+        .then((result) => {
           console.log(result);
-        }
-      );
+        });
```

Comment:
Nit: We should probably not encourage people to ignore errors. Logging it is the least we could do.
```suggestion
        })
        .catch(console.error);
```

Reply:
ee70c19